src/App.tsx

```diff
@@ -32,15 +32,15 @@ export default function App() {
     })
 
     // only use the api info's bbox until any other bounding box was chosen. Is this too messy?
-    const chooseBoundingBox = function (infoBbox: Bbox, pathBbox: Bbox, shouldUseInfoBbox: boolean) {
-        if (shouldUseInfoBbox && pathBbox.every(num => num !== 0)) {
+    const chooseBoundingBox = function (infoBbox: Bbox, shouldUseInfoBbox: boolean, pathBbox?: Bbox) {
+        if (shouldUseInfoBbox && pathBbox && pathBbox.every(num => num !== 0)) {
             setUseInfoBbox(false)
             return pathBbox
         } else if (shouldUseInfoBbox) return infoBbox
-        return pathBbox
+        return pathBbox || [0, 0, 0, 0]
```

Comment:
btw: we use this BBox as the "invalid case" here and elsewhere, but strictly speaking this is a perfect valid bbox :)

Still should not make a problem as this bbox is unrealistically small and points to nowhere

Reply:
The problem here was that the path coming from the api doesn't have a bbox property for routes with the same start and end point. Mapbox can actually handle the bbox with no area. 

The rest of the code relies on the bbox being at least present.

Reply:
> The problem here was that the path coming from the api doesn't have a bbox property for routes with the same start and end point. 

Do you mean we include the bbox in the response of /route and do not include it when two coordinates are identical? Or do you mean with "api" our JS code?

Reply:
Oh, indeed. This is a bug I think.

Reply:
https://github.com/graphhopper/graphhopper/issues/2277